src/components/AddItems.js

```diff
@@ -1,9 +1,101 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import db from '../firebase/firebase';
+import './AddItems.css';
+
+const firebase = require('firebase/app');
+require('firebase/firestore');
+
+const userToken = 'token';
 
 const AddItems = () => {
+  const [formData, setFormData] = useState({
+    itemName: '',
+    timeFrame: 7,
+    lastPurchased: null,
+    userToken: userToken,
+    dateCreated: new Date(),
+  });
+
+  //references the doc we are updating and changing
+  const itemsRef = db.collection('items');
+
+  // handle change of each form input, set state
+  const updateInput = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  // submits state to database
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    itemsRef
+      .add(formData)
+      .then(function () {
+        console.log('submitted!');
+        alert('submitted');
+      })
+      // catches & logs any errors
+      .catch(function (error) {
+        console.error('error updating count!', error);
```

Comment:
I think this may be an old copy-paste. I would suggest a more meaningful error message here.

Reply:
Thank you! It is from before. I will update it right now!